crates/db_schema/src/lib.rs

```diff
@@ -53,13 +62,29 @@ pub enum CommentSortType {
   Old,
 }
 
-#[derive(EnumString, Display, Debug, Serialize, Deserialize, Clone, Copy, PartialEq, Eq)]
+#[derive(
+  EnumString, Display, Debug, Serialize, Deserialize, Clone, Copy, PartialEq, Eq, DbEnum,
+)]
+#[ExistingTypePath = "crate::schema::sql_types::ListingTypeEnum"]
+#[DbValueStyle = "verbatim"]
 pub enum ListingType {
   All,
   Local,
   Subscribed,
 }
 
+#[derive(
+  EnumString, Display, Debug, Serialize, Deserialize, Clone, Copy, PartialEq, Eq, DbEnum,
+)]
+#[ExistingTypePath = "crate::schema::sql_types::RegistrationModeEnum"]
+// TODO snake case only for this one seems unecessary
+#[DbValueStyle = "snake_case"]
```

Comment:
There's no reason this needs to be coerced to snake_case, both rust and postgres can use `UpperCase`, and most client are going to copy-paste either this or lemmy-js-client anyway.

Reply:
Iirc I didnt want to make a db migration just to change the case.  Can definitely change it as youre already doing other migrations.

Reply:
Done.